docs/utilities/recipes.md

```diff
@@ -0,0 +1,213 @@
+
+Shaped and shapeless crafting
+-----------------------------
+Within this section we will take a closer look on the differences between defining a shaped and a shapeless crafting recipe.
+
+### Shaped crafting
+Shaped recipes require the `pattern` and `key` keywords. A pattern defines the slot an item must appear in using placeholder characters. You can choose whatever character you want to be a placeholder for an item. Keys on the other hand define what items are to be used instead of the placeholders. A key is defined by a placeholder character and the item. 
+Additional the type `forge:ore_dict` may be added. This defines the item beeing part of the [`OreDictionary`][OreDictionary] and can for example be used when it doesn't matter which copper ore is used to produce a copper ingot. In this case the `ore` tag has to be used instead of the `item` tag to define the item. There are [many more][Wiki] of these types which can be used here and you can even register your own.
+The `data` tag is a optional and used to define the metadata of a block or item.
+
+### Shapeless crafting
+A shapeless recipe doesn't make use of the `pattern` and `key` keywords. 
+
+To define a shapeless recipe, you have to use the `ingredients` list. It defines which items have to be used for the crafting process and can also make use of the additional type `forge:ore_dict` and it's functionality as described above. There are [many more][Wiki] of these types which can be used here and you can even register your own. It is even possible to define multiply instances of the same item which means multiply of these items have to be in place for the crafting recipe to take place. 
+
+!!! note
+
+    While there is no limit on how many ingredients your recipe requires the vanilla crafting table does only allow 9 items to be placed for each crafting recipe.
+
+The following example shows how an ingredient list looks like within JSON.
+
+```json
+    "ingredients": [
+        {
+            "type": "forge:ore_dict",
+            "ore": "minecraft:diamond"
+        },
+        {
+            "item": "minecraft:nether_star"
+        }
+    ],
+```
+
+### Smelting
+To define a recipe for the furnace, you have to use `GameRegistry.addSmelting(input, output, exp);` as the smelting recipes are currently not JSON based.
+
+### Special behaviour of the `data` tag
+It is strongly advised to use the `data` tag to define the metadata of your items and blocks. Any item which uses `setHasSubtypes(true)` requires the use of the data field. When it is not used within the ingredients or keys, it will mean any metadata of this item will be accepted, for example: Not defining the data of a sword means even a half broken sword will be accepted for the crafting recipe!
+
+Patterns
```

Comment:
This seems out of order. Shouldn't these next two sections be in the shaped recipe section?

Reply:
Well, I just put them to the results section as it matched quite nice, but thinking of it... As they don't really apply to the shapeless sections it makes more sense to move them to the shaped section.